src/Microsoft.ML.Data/EntryPoints/TransformModelImpl.cs

```diff
@@ -17,42 +18,45 @@ namespace Microsoft.ML.Runtime.EntryPoints
     /// the initial schema, together with the sequence of transforms applied to that
     /// schema.
     /// </summary>
-    public sealed class TransformModel : ITransformModel
+    [BestFriend]
+    internal sealed class TransformModelImpl : TransformModel
     {
         // The cached schema of the root of the _chain.
         private readonly Schema _schemaRoot;
 
-        // This contains the transforms to save instantiated on an IDataView with
-        // appropriate initial schema. Note that the "root" of this is typically either
-        // an empty IDataView or a BinaryLoader with no rows. However, other root
-        // types are possible, since we don't insist on this when loading a model
-        // from a zip file. However, whenever we save, we force a BinaryLoader to
-        // be serialized for the root.
+        /// <summary>
+        /// This contains the transforms to save instantiated on an <see cref="IDataView"/> with
+        /// appropriate initial schema. Note that the "root" of this is typically either
+        /// an empty <see cref="IDataView"/> or a <see cref="BinaryLoader"/> with no rows. However, other root
+        /// types are possible, since we don't insist on this when loading a model
+        /// from a zip file. However, whenever we save, we force a <see cref="BinaryLoader"/> to
+        /// be serialized for the root.
+        /// </summary>
         private readonly IDataView _chain;
 
         /// <summary>
         /// The input schema that this transform model was originally instantiated on.
         /// Note that the schema may have columns that aren't needed by this transform model.
-        /// If an IDataView exists with this schema, then applying this transform model to it
+        /// If an <see cref="IDataView"/> exists with this schema, then applying this transform model to it
         /// shouldn't fail because of column type issues.
         /// REVIEW: Would be nice to be able to trim this to the minimum needed somehow. Note
         /// however that doing so may cause issues for composing transform models. For example,
         /// if transform model A needs column X and model B needs Y, that is NOT produced by A,
         /// then trimming A's input schema would cause composition to fail.
```

Comment:
want to put this as a simple comment, outside of the summary node?

Reply:
I'm not sure. It's a summary comment on an internal class, so I'm not too excited about it? I mean, as you see I've fixed it insofar as I have an actual reference to the type now, which is surely an improvement, but beyond that I'm not too excited?